src/services/programService/index.js

```diff
@@ -318,7 +321,7 @@ export const listCountries = async (jwt = null) => {
 		});
 };
 
-export const listDataCenters = async (shortName, jwt) => {
+export const listDataCenters = async (shortName, jwt, id) => {
 	const url = urljoin(PROGRAM_SERVICE_HTTP_ROOT, `/datacenters`);
```

Comment:
`const url = urljoin(PROGRAM_SERVICE_HTTP_ROOT, 'datacenters');`
should be ok as this - url join should handle the slash and there's no need to use a template string because there's no variables being interpolated